django/webscraper/models.py

```diff
@@ -11,63 +10,32 @@ class Pet(models.Model):
         ("m", "male"),
         ("f", "female"),
     )
-    no = models.CharField(max_length=20)
-    name = models.CharField(max_length=100)
-    breed = models.CharField(max_length=100)
-    age = models.IntegerField()
-    gender = models.CharField(max_length=1, choices=SEX)
-    weight = models.IntegerField()
-    status = models.CharField(max_length=50)
-    date_in = models.DateField()
-    date_out = models.DateField()
-    address_found = models.CharField(max_length=100)
-    box = models.CharField(max_length=50)
-    group_name = models.CharField(max_length=50)
-    group_link = models.URLField()
-    link = models.URLField()
-
-
-class WebscrappingProcess(models.Model):
+    no = models.CharField(max_length=20, null=True)
```

Comment:
Is this change required / intentional? It may change DB schema.

Reply:
This is due to the possible errors when scraping animals. During scraping i encountered errors and the easiest fix was to allow null values.